gremlin-javascript/src/main/javascript/gremlin-javascript/lib/driver/connection.js

```diff
@@ -96,25 +92,61 @@ class Connection {
     this.isOpen = false;
     this.traversalSource = options.traversalSource || 'g';
     this._authenticator = options.authenticator;
+
+    this._timeoutAutoReconnectionInterval = 500;
+
+    this._pingEnabled = this.options.pingEnabled === false ? false : true;
+    this._pingIntervalDelay = this.options.pingInterval && this.options.pingInterval > 0 ? this.options.pingInterval : pingIntervalDelay;
```

Comment:
Nit: this condition can be simplified to `this.options.pingInterval || pingIntervalDelay`
(see https://github.com/airbnb/javascript#comparison--unneeded-ternary)